test/unit_system_builder_test.rb

```diff
@@ -109,4 +109,24 @@ class Measured::UnitSystemBuilderTest < ActiveSupport::TestCase
     assert_equal (2240), measurable.unit_system.unit_for!(:long_ton).conversion_amount
     assert_equal "lb", measurable.unit_system.unit_for!(:long_ton).conversion_unit
   end
+
+  test "#cache is nil by default" do
+    measurable = Measured.build do
+      unit :m
+      unit :km, value: "1000 m"
+    end
+
+    refute_predicate measurable.unit_system, :cached?
```

Comment:
minor nit: `"#cache is nil by default"` but we are testing for a false predicate. Even though `nil` is false-y in Ruby, shall we just test for it being exactly `nil`?

Reply:
This actually tests the `true`/`false`. I fixed the test description, not the assertion.